server/src/instant/util/logging_exporter.clj

```diff
@@ -54,7 +54,8 @@
                   "detailed_query"
                   "detailed_patterns"
                   "detailed_tx_steps"
-                  "process_id"})
+                  "process_id"
+                  "query"})
```

Comment:
Removes `query` from `datalog-query-reactive!`, which generates a lot of text, and `instaql/permissioned-query` (but we get the same query in `instaql/query-nested`).